backstory/src/main/java/com/google/sps/story/StoryManager.java

```diff
@@ -30,4 +30,14 @@
    * @param factory StoryManagerRequestFactory to use for HttpRequests.
    */
   public void setRequestFactory(StoryManagerRequestFactory factory);
+
+  /**
+   * Cycles selectedUrlIndex to a backup url.
+   */
+  public void cycleUrl();
```

Comment:
Can you explain when this might be needed?

Reply:
If the container throws an error, the backstory servlet can switch to another container.